cluster/examples/kubernetes/ceph/object-openshift.yaml

```
@@ -93,6 +93,11 @@ spec:
     # priorityClassName: my-priority-class
    #zone:
     #name: zone-a
+   #service:
+    #  key: value
+    # A key/value list of labels for rgw service
+     #annotations
+       #service.beta.openshift.io/serving-cert-secret-name: <name of TLS secret automatic generation>
```

let's try to keep a consistent comment style for manifests. Comment out whole blocks at a time such that if someone with a fancy code editor like VS code were to uncomment the whole block, they could still use the manifest. Comments are double commented and comment markers (hashes) have a space after them.

```suggestion
   # zone:
   #   name: zone-a
   # # Configuration for the Kubernetes Service created for the RGW cluster
   # service:
   #   # A key-value list of annotations to apply to the RGW Service
   #   annotations:
   #     service.beta.openshift.io/serving-cert-secret-name: <name of TLS secret automatic generation>
```